src/components/Filters/index.tsx

```diff
@@ -0,0 +1,74 @@
+import Button, { ButtonSize, ButtonVariants } from 'components/shared/Buttons';
+import InputField, { InputVariants } from 'components/shared/InputFields';
+import React, { useState } from 'react';
+
+const Filters: React.FC = () => {
+  const [condition, setCondition] = useState('');
+  const [category, setCategory] = useState('');
+  const [minPrice, setMinPrice] = useState('');
+  const [maxPrice, setMaxPrice] = useState('');
+
+  const submitFilters = () => {
+    const filters = {
+      condition,
+      category,
+      price: { min: minPrice, max: maxPrice },
+    };
+    console.log(filters);
```

Comment:
Remove console.log

Reply:
Ah I guess this is a feature to implement in the near future, it's ok, leave it like this